templates/rubocop.rb

```diff
@@ -48,4 +48,6 @@
 end
 
 run "bundle binstubs rubocop"
-run "bin/rubocop -A"
+
+count_of_files = Dir["**/*"].length
```

Comment:
Maybe we add only ruby files from `app`, `lib`, `spec`, `test` and `config` directories?